spec/intl.html

```diff
@@ -35,42 +35,42 @@ <h1>Intl[ @@toStringTag ]</h1>
     <h1>Constructor Properties of the Intl Object</h1>
 
     <emu-clause id="sec-intl.locale-intro">
-      <h1>Intl.Locale (...)</h1>
+      <h1>Intl.Locale ( . . . )</h1>
```

Comment:
It in necessary to put the space between the dots too?

Reply:
According to the ecmarkup linter, yes.

Reply:
It brings this spec into alignment with ECMA 262 ([example](https://tc39.es/ecma262/#sec-constructor-properties-of-the-global-object)), so I'm in favor.